infra/terraform/file_transfer.go

```diff
@@ -56,12 +56,13 @@ func (t *terraform) makeRemoteCommand(fileUrl, command string) (string, error) {
 			do test -f /var/lib/bootstrap_complete && break || \
 			echo Waiting for bootstrap to complete && sleep 15 ; \
 		done &&  \
-		echo Cleaning up && rm -rf %[1]s/installer/* && 
-		echo Downloading installer %[5]s to %[3]s ... && %[2]s && \
+		echo Cleaning up && rm -rf %[1]s/installer/* && \
+        	if [[ ! -f %[3]s ]]; then echo Downloading installer %[5]s to %[3]s ... ; %[2]s; fi && \
```

Comment:
Do not download installer, because it can be exists after executing client's bootstrap. Inside `user_data` for example.